db/migrate/20180601145643_add_encrypted_recovery_code_digest_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddEncryptedRecoveryCodeDigestToUser < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :encrypted_recovery_code_digest, :string
```

Comment:
should this be non-null and/or have a default value of `""` like `encrypted_password_digest`?